src/Http/Http.Abstractions/ref/Microsoft.AspNetCore.Http.Abstractions.netcoreapp.cs

```diff
@@ -250,6 +250,18 @@ public static partial class HttpMethods
         public static bool IsPut(string method) { throw null; }
         public static bool IsTrace(string method) { throw null; }
     }
+    public static partial class HttpProtocol
+    {
+        public static readonly string Http10;
```

Comment:
We considered `Http1_0` in the API review, but decided we already have precedence for this convention: https://docs.microsoft.com/en-us/dotnet/api/system.net.httpversion?view=netframework-4.8